CI/macos/appcast_download.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+import requests
+import xmltodict
+
+
+def download_build(url):
+    print('Downloading', url)
+    filename = url.rpartition('/')[2]
+    r = requests.get(url)
+    if r.status_code == 200:
```

Comment:
Should handle any status except `200 OK`, otherwise the script silently continues, leading to undefined state.